spec/acceptance/default_parameters_spec.rb

```diff
@@ -9,7 +9,6 @@ class { 'java':
 
       class { 'postgresql::globals':
         manage_package_repo => true,
```

Comment:
Do we need to manage the package repo or is the OS version sufficient? Looking at the `postgresql::globals` code it then uses the OS default version so I'd think you can drop the entire `class { 'postgresql::globals': }` block.